core/src/options/from_variant.rs

```diff
@@ -51,7 +51,7 @@ impl ParseAttribute for FromVariantOptions {
 
 impl ParseData for FromVariantOptions {
     fn parse_field(&mut self, field: &Field) -> Result<()> {
-        match field.ident.as_ref().map(|i| i.as_ref()) {
+        match field.ident.as_ref().map(|i| i.to_string().as_ref()) {
```

Comment:
Should this line be using `as_str`?

Reply:
Technically it works fine but for consistency reasons, yeah, you're right.